sunbeam/hooks.py

```diff
@@ -85,3 +104,6 @@ def configure(snap: Snap) -> None:
     :return: None
     """
     setup_logging(snap.paths.common / 'hooks.log')
+    logging.info("Running configure hook")
+
+    _update_default_config(snap)
```

Comment:
This should be called in the install hook as well, to ensure that the config parameters are set when referenced by the later code. Currently, they won't be set.

Reply:
The config parameters are used only in subsequent commands like init, status and not during install. So once snap is installed, the config params are available. I am not sure if we really need it again in install hook. 

``
$ sudo snap get microstack control-plane
Key                  Value
control-plane.cloud  microk8s
control-plane.model  openstack
``

Reply:
Ignore my previous comment, I understood what you mean. Updated in latest push, thanks!